Exercise_8/.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
-    "extends": "airbnb-base"
+    "extends": "airbnb-base",
+    "rules": "line-break: "off",
```

Comment:
Is this line ok for you? :) I believe, even your IDE should complain about it because it is not valid JSON even. And "rules" is an array.